app/scripts/modules/core/src/application/config/managedResources/ManagedResourceConfig.tsx

```diff
@@ -69,7 +69,7 @@ const ManagedResourceConfig = ({ application }: IManagedResourceConfigProps) =>
     setPauseFailed(false);
     logClick('Pause Management', application.name);
 
-    ManagedWriter.pauseResourceManagement(application.name)
+    ManagedWriter.pauseApplicationManagement(application.name)
       .then(() => {
         setPaused(true);
         application.managedResources.refresh();
```

Comment:
we should probs update this and L89 to use the `forceRefresh` arg just like your usage of `refresh` for status indicators.

Reply:
can't argue with that (will do)